test/ussd_nurse_rapidpro.test.js

```diff
@@ -51,12 +51,75 @@ describe('app', function() {
 
         describe('state_start', function() {
 
+            describe('user registered on nurseconnect', function(){
+                it('should go to state_registered_menu', function(){
+                    return tester
+                    .setup.user.addr('27820001003')
+                    .inputs(
+                        {session_event: 'new'}  // dial in
+                    )
+                    .check.interaction({
+                        state: 'state_registered_1',
+                        reply: [
+                                "Welcome back to NurseConnect. Do you want to:",
+                                "1. Help a friend sign up",
```

Comment:
or maybe `fellow nurse` just to be clear

Reply:
The SxD defined the words in the flow. I cant change it, might clash with the content team